### PR TITLE
Speed improvements

### PR DESCRIPTION
**Pull request by abcdan**
I made `nocode` a lot faster, like a lot-lot.

### CODE REVIEW

**Comment by Hocuri** [comment]
Could you add some measurements for this?

**Comment by PathToLife** [comment]
no comment

**Comment by PathToLife** [comment]
no comment

**Comment by abcdan** [comment]
> Could you add some measurements for this?

It went from `    `  to  `    ` in speed!

**Comment by Hocuri** [comment]
@abcdan tbh, I'm not sure that's worth it. I mean, it was pretty fast before, and code readability went down quite a bit

**Review by vadimfilimonov — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,202 +0,0 @@
-
-                                 Apache License
```
Comment (PathToLife) [inline LICENSE:2]: Legal wants us to keep this.

**Review by PathToLife — APPROVED** [review]
Other than that legal stuff, which we can deal with later. LGTM

**Pull request closed without merging**